### PR TITLE
Feature/cli syntax updates

### PR DESCRIPTION
**Pull request by jenncarneiro**
# Critical Changes

# Changes
CLI syntax updates to recipes in these folders :
OBF
OSC

# Issues Closed

### CODE REVIEW

**Comment by jenncarneiro** [comment]
@prescod Got an error running the MembershipSchemaAndBenefits.recipe.yml :

`Error: Problem rendering field __pricebookentry_from_salesforce:
 Cannot find definition for: SOQLDataset
 near /Users/jennifercarneiro/Documents/Minisprint/snowfakery_samples/OSC/MembershipSchemaAndBenefits.recipe.yml:107`

Not sure if there is just something I am missing or if it's an actual issue - over my head but I figured someone should check it out. Tag - you're it!

**Comment by acrosman** [comment]
The project that maintains that package is also working on new recipes. I work with the person who was doing that during the sprint. I'll see if she has suggestions for just replacing this with something newer.

**Comment by eehjunggnujhee** [comment]
@jenncarneiro @acrosman The changes are all minor spelling fixes.  The recipe requires the installation of OSC project called, https://github.com/SFDO-Community-Sprints/MembershipSchemaAndBenefits
I will look into it this weekend!

File: snowfakery_samples/OBF/OBF_basic.recipe.yml [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
-# Run this to generate data in a QA scratch org:
-# cci task run generate_and_load_from_yaml --generator_yaml ../snowfakery/OBF_basic.recipe.yml --num_records 10 --num_records_tablename outfunds__Funding_Program__c --org qa
-
+#
+### --------------- [ CLI RUN ] ------------------------------ ###
+# Run this to generate data 10 times in a scratch org called dev:
+# cci task run snowfakery --recipe snowfakery_samples/OBF/OBF_basic.recipe.yml --run_until_recipe_repeated 10 --org dev
+#
 # Run this to generate records in an output:
 # snowfakery ./OBF_basic.recipe.yml --output-format=json --output-file=foo.json
```
Comment (eehjunggnujhee) [inline snowfakery_samples/OBF/OBF_basic.recipe.yml:15]: nice work @jenncarneiro a few minor directory changes here
from
`snowfakery ./OBF_basic.recipe.yml --output-format=json --output-file=foo.json`
to
`snowfakery snowfakery_samples/OBF/OBF_basic.recipe.yml --output-format=json --output-file=snowfakery_samples/temp/foo.json`

**Review by eehjunggnujhee — COMMENTED** [review]
@jenncarneiro  The recipes ran nicely. Thanks for your hard work!
`snowfakery_samples/OSC/MembershipSchemaAndBenefits_PricebookProducts.recipe.yml`
`snowfakery_samples/OBF/OBF_basic.recipe.yml`

This recipe requires two pre-config:
snowfakery_samples/OSC/MembershipSchemaAndBenefits.recipe.yml
1. an org connection to extract existing data and use in the recipe.
2. install [MembershipsandBenefitsSchema](https://github.com/SFDO-Community-Sprints/MembershipSchemaAndBenefits) and assign Manage_Benefits perm set.
3. Enable field level permission on OpportunityLineItem.Membership__c.

After these, the recipe ran as expected. We will be able to approve the PR after reviewing the suggested updates

**Review by eehjunggnujhee — APPROVED** [review]
Merging the change

**Pull request merged**